test/arrays/skyline/spawnPromoteBug.good

```diff
@@ -0,0 +1 @@
+spawnPromoteBug.chpl:10: error: creating an array of arrays using a for- or forall-expression is not supported, except when using a for-expression over a range
```

Comment:
I'm having trouble understanding why this error message is reasonable? `p.stdout.lines()` isn't producing an array of arrays. Seems to me this should be a .bad file and that we would expect this program to function as-is.

(I think the case here is that the compiler is getting tripped up by `p.stdout.lines()` being an `ItemReader`. We could change it to a regular iterator.)

Reply:
I think that line 9 is promoting `f` over the two array elements in the argument, so `p` is an array of `subprocess` records, so `p.stdout` is an array of `channel`s. So `p.stdout.lines()` is an array of iterator calls, which flatten into arrays.

I think the error message is not great, but technically correct.

Reply:
Yeah, David beat me to it, but that was my interpretation as well.